config/locales/en.yml

```diff
@@ -430,19 +437,20 @@ en:
       search_on_this_page: "Search on this page"
       no_results: "No results"
       download: "Download:"
+
       
-  # activerecord:
-    # attributes:
-      # client:
-        # ssn: "Social security number"
-        # dob: "Date of birth"
-        # num_in_household: "Number in household"
-        # num_of_dependants: "Number of dependants"
-    # errors:
-    #   models:
-    #     client:
-    #       attributes:
-    #         ssn:
-    #         dob:
-    #         num_of_dependants:
-    #         num_in_household:
```

Comment:
I tried it, and i could never figure it out. You're right, I should have just kept it commented out...